packages/functions/src/dal/EmailNotificationsDataAccessLayer.ts

```diff
@@ -2,10 +2,10 @@ import * as admin from "firebase-admin";
 import { Collections } from "@zm-blood-components/common";
 
 export type EmailMessage = {
-  to: string[];
+  to: string | string[];
```

Comment:
Why more than 1 type?

Reply:
This is what the framework support, either one email or an array of emails. both work